java/source/create.rst

```diff
@@ -0,0 +1,134 @@
+======================
+Creating arrow objects
+======================
+
+A vector is the basic unit in the java arrow columnar format.
+Vectors are provided by java arrow for the interface FieldVector that extends ValueVector.
```

Comment:
Can we link to Javadocs? Is there an extension that can automate this in the same way intersphinx does for Sphinx docs?

Reply:
What is the distinction between FieldVector and ValueVector?

Reply:
Maybe we should explain that vectors in Java are intended to be mutable, since that will also be foreign to users of other Arrow libraries.

Reply:
Changed